src/integrationTest/java/org/opensearch/security/SearchOperationTest.java

```diff
@@ -371,6 +373,16 @@ public class SearchOperationTest {
             USER_ALLOWED_TO_PERFORM_INDEX_OPERATIONS_ON_SELECTED_INDICES,
             USER_ALLOWED_TO_CREATE_INDEX
         )
+        .nodeSettings(
```

Comment:
This flag is noop if plugin is not installed, right? I think what we could do is to enhance (or add new) the GA which does full plugin install test to have security and opentelemetry plugin installed together, @peternied does it sound right to you?

Reply:
yes @reta you are right. but this would atl east create the Traceable wappers.

Reply:
> yes @reta you are right. but this would at least create the Traceable wrappers.

how? it will use NOOP tracer

Reply:
Oh, yes. After this recent change tracer:isTracingEnabled, it wont.

Reply:
I've moved this flag to its own class - but I was not able to reproduce a failure with the other changes in this CR reverted.  

Please pull the latest changes from your fork and test locally until the test can fail if you'd reverted the other changes - its possible I've got the feature flag configured incorrectly.